============================= test session starts ==============================
platform linux -- Python 3.7.11, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/devops/entrega/homework, configfile: pytest.ini
collected 1 item / 1 error

==================================== ERRORS ====================================
_________________ ERROR collecting app/tests/argparse_test.py __________________
../../../.local/share/virtualenvs/homework-F_5ItYF7/lib/python3.7/site-packages/_pytest/python.py:578: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../../../.local/share/virtualenvs/homework-F_5ItYF7/lib/python3.7/site-packages/_pytest/pathlib.py:524: in import_path
    importlib.import_module(module_name)
/usr/local/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1006: in _gcd_import
    ???
<frozen importlib._bootstrap>:983: in _find_and_load
    ???
<frozen importlib._bootstrap>:967: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:677: in _load_unlocked
    ???
../../../.local/share/virtualenvs/homework-F_5ItYF7/lib/python3.7/site-packages/_pytest/assertion/rewrite.py:161: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../../../.local/share/virtualenvs/homework-F_5ItYF7/lib/python3.7/site-packages/_pytest/assertion/rewrite.py:354: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/local/lib/python3.7/ast.py:35: in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
E     File "/home/devops/entrega/homework/app/tests/argparse_test.py", line 20
E       assert
E            ^
E   SyntaxError: invalid syntax
=========================== short test summary info ============================
ERROR tests/argparse_test.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.22s ===============================
